### ISSUE TITLE
mGWT is Carousel not working with Chrome V71.0 with touch device

### ISSUE THREAD

**Issue by Vadirajachar** [mgwt/mgwt#290]
For any touch and drag gesture for that matter, it has stopped working. This includes pull-to-refresh, scroll widget and so on. 

Is there any quick work-around which I can use?

**Comment by schteff** [comment]
We're experiencing problems with newer Chrome on mobile here to (on newer Android versions). Some taphandlers and some (not all!) drag/touch handlers don't work. Trying to identify the issue now.

**Comment by schteff** [comment]
We were hit by this issue and temporary fixed it with min-height: 0; on an inner flex item. 

https://bugs.chromium.org/p/chromium/issues/detail?id=927066

Also I had to change one TapHandler to a ClickHandler in one place, still using some TapHandlers with no issue in other places in the app though.

**Comment by waynedyck** [comment]
@schteff thanks for the link. What a read. I will try and take a look at this over the next couple of days.

**Comment by schteff** [comment]
I'm not sure what we experienced is the same that @Vadirajachar reported. 

When I made the fix in our app i saw that we still have fixes for this old issue in the same place:
https://bugs.chromium.org/p/chromium/issues/detail?id=502422
(which won't be fixed)
![image](https://user-images.githubusercontent.com/2564884/52397233-fdad8c80-2ab4-11e9-8228-ae475b2304c6.png)

The affected view in our app is quite niche and has multiple inner flex panels that scroll in sync with eachother both horizontally and vertically. 
![image](http://g.recordit.co/o3lRDVrkGn.gif)

Just giving some context.
